ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/HeartbeatAgent.java

```diff
@@ -223,6 +223,9 @@ private void processHeartbeats(final long windowStart, final long windowEnd) {
           }
           return status;
         });
+        for (HostStatusListener listener : hostStatusListeners) {
```

Comment:
is this related to the single host lag reporting case?

Reply:
So previously, the heart beat agent was not notifying the lag agent?

Reply:
Yes, if there are no heartbeats received, it will still update the status of known hosts (localhost) and that needs to be reported to the listeners.